### DISCUSSION TITLE
Copilot is down. How to debug and see anything? Total silent failure.

### DISCUSSION THREAD

**Discussion by cottrell** [community/community#106519] [Copilot Conversations]
### Select Topic Area

Bug

### Body

In vscode.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @cottrell ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by cottrell** [reply]
FYI the help somewhere directed me specifically to *discussions* which I thought was weird but followed the instructions anyway. I'll post again if I re-find the documentation link.

**Comment by cottrell** [comment]
Maybe:  WARN Settings pattern "GitHub.copilot.manageExtension" doesn't match any settings

Might have started working again with very bad latency.
